angular.json

```diff
@@ -119,6 +121,10 @@
           }
         }
       }
-    }},
-  "defaultProject": "OnlineShopFE"
-}
+    }
+  },
+  "defaultProject": "OnlineShopFE",
+  "cli": {
+    "analytics": false
+  }
+}
```

Comment:
Usually, you will need to have an empty line at the end of the file. You can also see that github warns you about this